src/modals/AddItemModal.tsx

```diff
@@ -73,7 +72,7 @@ const AddItemModal: FC<AddItemModalProps> = ({ tooltipLabel, onRefresh }) => {
         genericErrorToast(err, toast);
       });
     setItemData({
-      name: "",
+      name: itemRef.current?.value,
```

Comment:
We are resetting fields after successfully submission, that's why the name should be empty 

```suggestion
      name: "",
```